prow/cmd/gangway/main.go

```diff
@@ -50,10 +51,10 @@ import (
 const serviceNameForHealthCheckGrpc = ""
 
 type options struct {
-	client         prowflagutil.KubernetesOptions
-	github         prowflagutil.GitHubOptions
-	port           int
-	cookiefilePath string
+	client           prowflagutil.KubernetesOptions
+	github           prowflagutil.GitHubOptions
+	port             int
+	cookiefilePath   string
```

Comment:
Did gofmt add whitespace? (Was there an insert then delete?)

Reply:
Odd. Not sure what happened there. Fixed locally.